elide-contrib/elide-dynamic-config-helpers/src/main/java/com/yahoo/elide/contrib/dynamicconfighelpers/DynamicConfigSchemaValidator.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2020, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide.contrib.dynamicconfighelpers;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.github.fge.jsonschema.core.exceptions.ProcessingException;
+import com.github.fge.jsonschema.core.report.ProcessingReport;
+import com.github.fge.jsonschema.main.JsonSchema;
+import com.github.fge.jsonschema.main.JsonSchemaFactory;
+
+import lombok.extern.slf4j.Slf4j;
+
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.Reader;
+
+/**
+ * Dynamic Model Schema validation.
+ */
+@Slf4j
+public class DynamicConfigSchemaValidator {
+
+    private static final JsonSchemaFactory FACTORY = JsonSchemaFactory.byDefault();
+    private JsonSchema tableSchema;
+    private JsonSchema securitySchema;
+    private JsonSchema variableSchema;
+
+    public DynamicConfigSchemaValidator() {
+        tableSchema = loadSchema(Config.TABLE.getConfigSchema());
+        securitySchema = loadSchema(Config.SECURITY.getConfigSchema());
+        variableSchema = loadSchema(Config.VARIABLE.getConfigSchema());
+    }
+    /**
+     *  Verify config against schema.
+     * @param configType
+     * @param jsonConfig
+     * @return whether config is valid
+     * @throws IOException
+     * @throws ProcessingException
+     */
+    public boolean verifySchema(Config configType, String jsonConfig) throws IOException, ProcessingException {
+        ProcessingReport results = null;
+        boolean isSuccess = false;
+
+        switch (configType) {
+        case TABLE :
+            results = this.tableSchema.validate(new ObjectMapper().readTree(jsonConfig));
+            break;
+        case SECURITY :
+            results = this.securitySchema.validate(new ObjectMapper().readTree(jsonConfig));
+            break;
+        case VARIABLE :
+            results = this.variableSchema.validate(new ObjectMapper().readTree(jsonConfig));
+            break;
+        default :
+            log.error("Not a valid config type :" + configType);
+            return false;
```

Comment:
A method like this should either return false or throw an exception but not both.  Otherwise the contract is confusing.